network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube-regrets-2022/fragments/petition_cta.html

```diff
@@ -0,0 +1,14 @@
+{% load static i18n %}
+<section class="tw-dark tw-my-7 medium:tw-mt-[120px] medium:tw-mb-[80px]">
+  <div class="container tw-max-w-3xl tw-mx-auto">
+    <div class="tw-flex tw-flex-col tw-justify-center tw-items-center tw-text-center">
+      <svg class="tw-w-[45px] tw-h-[42px] tw-mb-3" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 64 56.93" aria-hidden="true"><path d="M59.16,0H4.84C2.17,0,0,2.17,0,4.84V40.65c0,2.67,2.17,4.84,4.84,4.84H15.42v9.61c0,1.55,1.81,2.4,3,1.4l13.15-11.01h27.59c2.67,0,4.84-2.17,4.84-4.84V4.84c0-2.67-2.17-4.84-4.84-4.84Z" fill="#fff"/><circle cx="22.78" cy="15.28" r="3.82"/><path d="M22.49,33.79c1.37-3.59,4.26-6.77,9.51-6.77s7.95,3.13,9.51,6.77" fill="none" stroke="#000" stroke-linecap="round" stroke-linejoin="round" stroke-width="6"/><circle cx="41.82" cy="15.28" r="3.82"/></svg>
+      <h2 class="tw-leading-none tw-text-4xl medium:tw-text-[3.25rem]">{% trans 'But it’s hard to know what really works.' %}</h2>
+      <p>{% trans "Let's call on YouTube to empower users with real, meaningful controls" %}</p>
+      <a class="btn btn-with-icon btn-white" href="https://foundation.mozilla.org/campaigns/youtube-user-controls">
```

Comment:
**Question** Is this link expected to be manipulated and lead to a 404 locally? E.g. for me it goes to: `https://foundation.mozilla.org/en/campaigns/youtube-user-controls/?_gl=1*k51lit*_ga*Mzc2ODIxMjIzLjE2NjIwNjE5MTQ.*_ga_4R07FEDXLC*MTY2MjA2OTE0My4yLjEuMTY2MjA3MzUyMS4wLjAuMA..` which is a 404.

Reply:
Good question. I believe most of these links actually don't exist yet and will be updated in the future iterations when we finalize content